dev/com.ibm.ws.concurrency.policy/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -0,0 +1,45 @@
+###############################################################################
+# Copyright (c) 2017 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+# -------------------------------------------------------------------------------------------------
+#CMVCPATHNAME com.ibm.ws.concurrency.policy/resources/OSGI-INF/l10n/metatype.properties
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+
+# configuration elements
+
+concurrencyPolicy=Concurrency Policy
+concurrencyPolicy.desc=A set of behaviors and constraints to be applied for tasks that are capable of asynchronous execution. For example maximum concurrency and maximum queue size.
+
+# attributes
+
+expedite=Expedite
+expedite.desc=Specifies a core number of tasks to aim to run concurrently by expediting requests to the global thread pool. This provides no guarantee that this many tasks will run concurrently. If the expedite value is updated while in use, the update goes into effect gradually as previous expedited and non-expedited requests complete.
+
+max=Maximum concurrency
+max.desc=Specifies the maximum number of tasks that can be running at any given point in time. The default is unbounded, up to Integer.MAX_VALUE. Maximum concurrency can be updated while tasks are in progress. If the maximum concurrency is reduced below the number of concurrently executing tasks, the update goes into effect gradually as in-progress tasks complete rather than causing them to be canceled.
+
```

Comment:
No, that sentence was meaning to explain that the default, being Integer.MAX_VALUE, is such a large value at that it can essentially be considered unbounded.  I will agree to remove the word unbounded which causes confusion here, and simply say, "The default is Integer.MAX_VALUE" along with your other proposed changes.

Reply:
That sounds good. Thank you for explaining!